app/models/section_edition.rb

```diff
@@ -39,9 +39,9 @@ class SectionEdition
 
   scope :with_slug_prefix, ->(slug) { where(slug: /^#{slug}.*/) }
 
-  index "section_id"
-  index "state"
-  index "updated_at"
+  index section_id: 1
+  index state: 1
+  index updated_at: 1
```

Comment:
I'm fairly happy to leave these magic numbers as they are. The Mongoid and MongoDB references about indexes talk about the index direction in terms of numbers (1 = ascending / -1 = descending). I've also explained in the commit note that these are set to 1 as that matches what was happening previously.